lib/source_readers/flat.rb

```diff
@@ -20,13 +20,14 @@ def self.resolve(target)
       new(target, files)
     end
 
-    attr_reader :metadata, :tests, :libraries
+    attr_reader :metadata, :tests, :libraries, :files
```

Comment:
I think it's going to get super-confusing for newcomers to this codebase that we call this "files" but it only includes files in the `files` directory, when the controls are stored in files also.

Should we call them "supplemental_files" or "data_files" or something of that nature?